src/Geno/C++/GUI/Widgets/TextEdit.h

```diff
@@ -266,4 +266,6 @@ class TextEdit
 
 	std::filesystem::path m_ActiveFilePath = {};
 
+	int m_CurrentLine = -1;
+
```

Comment:
Adding extra state to this class is bad if we can just make a function that fetches the current line instead.